versioned_docs/version-v3.0.x/extend/extend-apiml/api-mediation-infinispan.md

```diff
@@ -28,36 +28,52 @@ For more information on Infinispan replication and how to configure a replica in
 
 Configure Infinispan as a storage solution through the Caching service by setting the following configuration parameters in the `zowe.yaml`.
  
-* **`zowe.components.caching-service.storage.infinispan.initialHosts`**
+* **zowe.components.caching-service.storage.infinispan.initialHosts**
 
   This property specifies the list of cluster nodes (members). In case of multiple instances, the value for each Caching Service instance can be 
   either a list of all the members, separated by a comma, or just the replica. The format is `${haInstance.hostname}[${zowe.components.caching-service.storage.infinispan.jgroups.port}]`.
 
-* **`zowe.components.caching-service.storage.infinispan.persistence.dataLocation`**
+* **zowe.components.caching-service.storage.infinispan.persistence.dataLocation**
 
   The path where the service keeps its data files for the Infinispan Soft-Index Cache Store. 
-  The default value is `data`. If you run the Caching Service in HA and the instances use the same filesystem, you have to specify a different value of the data property for each instance. For more information, see the [Soft-Index File Store](https://infinispan.org/blog/2014/10/31/soft-index-file-store).
+  The default value is `data`. To run the Caching Service in HA, ensure that you apply the following configuration conditions:
 
+   - The value should be the same for each instance.
+   - The location should point to non-shared filesystem. Each instance requires unique storage.
+   - For more information, see the [Soft-Index File Store](https://infinispan.org/blog/2014/10/31/soft-index-file-store).
 
-* **`zowe.components.caching-service.storage.infinispan.persistence.indexLocation`**
+
+* **zowe.components.caching-service.storage.infinispan.persistence.indexLocation**
 
   The path where the service keeps its index data for the Infinispan Soft-Index Cache Store. 
-  The default value is `index`. If you run the Caching Service in HA and the instances use the same filesystem, you have to specify a different value of the index property for each instance. For more information, see the [Soft-Index File Store](https://infinispan.org/blog/2014/10/31/soft-index-file-store).
+  The default value is `index`. To run the Caching Service in HA, ensure that you apply the following configuration conditions:
+
+  - The value should be the same for each instance.
+  - The location should point to non-shared filesystem. Each instance requires unique storage.
+  - For more information, see the [Soft-Index File Store](https://infinispan.org/blog/2014/10/31/soft-index-file-store).
 
 
-* **`zowe.components.caching-service.storage.infinispan.jgroups.port`**
+* **zowe.components.caching-service.storage.infinispan.jgroups.port**
 
-  (OPTIONAL)The default value is `7098`. The port number used by Infinispan to synchronise data among cahing-service instances.
+  (Optional) The default value is `7600`. The port number is used by Infinispan to synchronise data among Caching Service instances.
```

Comment:
Since this is going with 3.2, I'd keep the 7098, I believe we need to update that in the `defaults.yaml`